### PR TITLE
Proto changes to receive and publish dormant time.

### PR DESCRIPTION
**Pull request by adarsh-zededa**
EVE can modify `timer.metric.interval` based on network connectivity. So EVE must report its version of dormant time
as part DeviceInfo message back to the controller so that it can modify the monitoring process accordingly.

Signed-off-by: adarsh-zededa <adarsh@zededa.com>

### CODE REVIEW

File: api/proto/config/devconfig.proto [hunk 1]
```diff
@@ -121,6 +121,11 @@ message EdgeDevConfig {
 
   // A list of bond interfaces (LAGs) aggregating physical network adapters.
   repeated BondAdapter bonds = 31;
+
+  // Dormant time configured from controller. This decides the duration to
+  // wait between metrics messages before marking an edge-node as inactive
+  // in the controller.
+  string config_dormant_time = 32;
```
Comment (eriknordmark) [inline api/proto/config/devconfig.proto:128]: Why do we need this in config?
We already have the configItem called timer.metric.interval and today the dormant time is 3x that timer by default.

What is missing is the dormant time in the info message.

**Review by eriknordmark — CHANGES_REQUESTED** [review]
In addition to the comment about the config side, please split the derived and vendored files into a separate commit from the proto file changes.

Are you also planning to add code to EVE-OS?

File: api/proto/config/devconfig.proto (hunk 1 above)
Comment (adarsh-zededa) [reply api/proto/config/devconfig.proto]: Removed from Config.

**Comment by adarsh-zededa** [comment]
> In addition to the comment about the config side, please split the derived and vendored files into a separate commit from the proto file changes.

Added 2 separate commits

> Are you also planning to add code to EVE-OS?

Yes @nordmark, I'll take up the task to add the necessary changes.

**Review by eriknordmark — APPROVED** [review]
LGTM

**Review by rvs — COMMENTED** [review]
LGTM

**Pull request merged**